### PR TITLE
Update to use latest api client from lisk-sdk - Closes #33

### LINKED ISSUE #33: Update faucet to support Lisk Core v3.0.0-beta.2
In order to release Lisk SDK 5.0.0 and Lisk Core v3.0.0-beta.2, we should update the lisk-faucet to support

- Update Transaction creation
- Update betanet configuration with networkIdentifier and other config changes
- Test signing and sending transaction using betanet configuration

### PR DESCRIPTION
**Pull request by shuse2**
### Description

- Update to use latest lisk-sdk (@liskhq/lisk-client)
- Package needs to be updated once sdk is published to the public repo

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN useradd lisk && \
     chown lisk:lisk -R /home/lisk
 USER lisk
 RUN cd /home/lisk/lisk-faucet && \
-    npm install
+    npm install --registry https://npm.lisk.io
```
Comment (ghost) [inline Dockerfile]: Isn't that for pre-release / devlopment only?

Comment (shuse2) [reply Dockerfile]: yes, as i mentioned on the PR, it needs to be updated once sdk is published

Comment (ghost) [reply Dockerfile]: Or we could default to not using any custom registry unless `$REGISTRY_URL` is set? See https://github.com/LiskHQ/lisk-core/blob/development/docker/Dockerfile#L22

Comment (shuse2) [reply Dockerfile]: Update not to use the registry =)

**Review by ghost — APPROVED** [review]

**Review by MichalTuleja — APPROVED** [review]
LGTM 👍

**Pull request merged**